_pages/channels/general.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: channel
+title: General
+channel: general
+permalink: /channels/general/
+parent: channels
+---
+
+A place for general discussion for IT tinkerers. Feel free to ask questions or post interesting industry news.
+
+## Rules
+
+* Please stay on topic, keep political posts in #politics, completely off-topic stuff in #random.
```

Comment:
>… completely non tech-related stuff in #random.

Because _off-topic_ is difficult to understand if we don’t know what the topic is.